starboard/nplb/player_test_fixture.h

```diff
@@ -95,6 +95,9 @@ class SbPlayerTestFixture {
   bool HasVideo() const { return video_dmp_reader_; }
 
   int64_t GetAudioSampleTimestamp(int index) const;
+
+  // Returns the number of buffers that contains |duration| worth of audio or
+  // video. Returns at minimum 1 buffer.
```

Comment:
Is there a specific reason we need to do that?

Reply:
Originally this was causing failures with the FLAC dmp files; I think |duration| ended up being <1 frame, so this returned 0 and some other code CHECK-failed.

However, I can't repro the issue anymore, so I've reverted this part of the change. If it happens again in the future, I'll send out a separate PR (with repro steps and more details).

Reply:
Now that I think about it more, it's possible that the failures stopped happening because I changed the dmp files a while back (commit 4, I think). I had added the rounding logic before that commit.